webcompat/form.py

```diff
@@ -48,32 +52,42 @@
 username_message = u'A valid username must be {0} characters long'.format(
     random.randrange(0, 99))
 
+desc_label = 'Please describe what was wrong <span class="wc-Form-required">' \
+    '*</span>'
+desc_message = u'An issue description is required.'
+
 problem_label = (u'What seems to be the trouble?',
                  '<span class="wc-Form-required">*</span>')
 url_label = u'Site URL <span class="wc-Form-required">*</span>'
 
-desc_default = u'''1. Navigate to: Site URL
-2. …
+steps_default = u'''1. I've tried to log in.
+2. I've filled out the form details.
+3. I clicked on the submit button.
+4. Nothing happened.
 
-Expected Behavior:
-
-Actual Behavior:
 '''
 
 
 class IssueForm(FlaskForm):
     '''Define form fields and validation for our bug reporting form.'''
     url = StringField(url_label,
                       [InputRequired(message=url_message)])
-    browser = StringField(u'Browser / Version', [Optional()])
+    browser = StringField(u'Is this information correct?', [Optional()])
     os = StringField(u'Operating System', [Optional()])
     username = StringField(u'Username',
                            [Length(max=0, message=username_message)])
-    description = TextAreaField(u'Give more details', [Optional()],
-                                default=desc_default)
+    description = StringField(desc_label,
+                              [InputRequired(message=desc_message)])
+
+    steps_reproduce = TextAreaField(u'How did you get there?', [Optional()],
+                                    default=steps_default)
     problem_category = RadioField(problem_label,
                                   [InputRequired(message=radio_message)],
                                   choices=problem_choices)
+    browser_test = RadioField(problem_label,
```

Comment:
@karlcow the main difference that I can see is the resulting `for="..."` value. 

![screen shot 2017-06-09 at 10 47 53 am](https://user-images.githubusercontent.com/4755432/26987771-aa7a29ca-4d01-11e7-9410-1fe2ab95f2c6.png)

my best guess is that the label is hand coded in the html to be able to hard code the `for` value. 
the `for="problem_type"` seems to be referenced by `problem_type` in form.py. I do not know exactly how it gets used end to end but seems to be significant and I felt it safer to preserve the existing way.

ex https://github.com/webcompat/webcompat.com/blob/master/webcompat/form.py#L205